pkg/deployments/tasks/20221202-timelock-authorizer/input/mainnet.ts

```diff
@@ -0,0 +1,313 @@
+  { actionId: Vault.actionId('Vault', 'setAuthorizer(address)'), newDelay: 30 * DAY },
+  {
+    actionId: SmartWalletChecker.actionId('SmartWalletChecker', 'allowlistAddress(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: VotingEscrowDelegationProxy.actionId('VotingEscrowDelegationProxy', 'setDelegation(address)'),
+    newDelay: 14 * DAY,
+  },
+];
+
+export const grantDelays: DelayData[] = [
+  {
+    actionId: BalancerTokenAdmin.actionId('BalancerTokenAdmin', 'mint(address,uint256)'),
+    newDelay: 30 * DAY,
+  },
+  {
+    actionId: Vault.actionId('ProtocolFeesCollector', 'withdrawCollectedFees(address[],uint256[],address)'),
+    newDelay: 30 * DAY,
+  },
+  {
+    actionId: GaugeController.actionId('GaugeController', 'add_gauge(address,int128)'),
+    newDelay: 14 * DAY,
+  },
+  {
+    actionId: GaugeController.actionId('GaugeController', 'add_gauge(address,int128,uint256)'),
+    newDelay: 14 * DAY,
+  },
+  {
+    actionId: GaugeAdder.actionId('GaugeAdder', 'addEthereumGauge(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: GaugeAdder.actionId('GaugeAdder', 'addPolygonGauge(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: GaugeAdder.actionId('GaugeAdder', 'addArbitrumGauge(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: GaugeAdder.actionId('GaugeAdder', 'addOptimismGauge(address)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: GaugeAdder.actionId('GaugeAdder', 'addGaugeFactory(address,uint8)'),
+    newDelay: 7 * DAY,
+  },
+  // BALTokenHolder.withdrawFunds(address, uint256) (veBAL BALTokenHolder)
+  // Note this actionId can't be pulled from the json file as the BALTokenHolder is not listed there.
+  { actionId: '0x79922681fd17c90b4f3409d605f5b059ffcbcef7b5440321ae93b87f3b5c1c78', newDelay: 7 * DAY },
+  {
+    actionId: Vault.actionId('Vault', 'setRelayerApproval(address,address,bool)'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: Vault.actionId(
+      'Vault',
+      'batchSwap(uint8,(bytes32,uint256,uint256,uint256,bytes)[],address[],(address,bool,address,bool),int256[],uint256)'
+    ),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: Vault.actionId('Vault', 'joinPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: Vault.actionId(
+      'Vault',
+      'swap((bytes32,uint8,address,address,uint256,bytes),(address,bool,address,bool),uint256,uint256)'
+    ),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: Vault.actionId('Vault', 'exitPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: Vault.actionId('Vault', 'manageUserBalance((uint8,address,uint256,address,address)[])'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: ComposableStablePool.actionId('ComposableStablePool', 'unpause()'),
+    newDelay: 7 * DAY,
+  },
+  {
+    actionId: WeightedPoolV2.actionId('WeightedPool', 'unpause()'),
+    newDelay: 7 * DAY,
+  },
```

Comment:
```suggestion
```

These pools are out of the pause window now so this permission is safe and doesn't need a delay.